apps/ios/src/Podfile.lock

```diff
@@ -19,6 +19,7 @@ PODS:
     - DoubleConversion
     - glog
   - glog (0.3.5)
+  - MicrosoftFluentUI (0.1.0)
```

Comment:
This specifies the dependencies in the test app but not for a consumer of the shared library and bundle?